docs/Config_Reference.md

```diff
@@ -3895,6 +3895,46 @@ host_mcu:
 #   (True sets CFG5 high, False sets it low). The default is True.
 ```
 
+## [safe_power]
```

Comment:
I think the documentation would be simpler if it always had a name on the config section (eg, `[safe_power myname]`).

Reply:
I will tighten up the documentation

/Lars